docs/docs/building-apps/03-migrations.md

```diff
@@ -2,7 +2,13 @@
 sidebar_position: 1
 ---
 
-# Pre-Upgrade Handling
+# Migrations
```

Comment:
Put the migrations to perform by a chain under `building-apps`, what do you think?
Then the migration, more SDK related is under the migration section.
I think it makes sense, but curious about your input.